src/public/js/follow-unfollowButton

```diff
@@ -1,19 +1,22 @@
 $('.toggle-button').click(function() {
     const userId = $(this).data('user');
-    const action = $(this).data('action');
     const button = $(this);
     const csrfToken = $('meta[name="csrf-token"]').attr('content');
+    const isUserFollowed = button.hasClass('btn-danger');
 
     $.ajax({
         type: 'POST',
-        url: '/users/' + userId + '/' + action,
+        url: '/users/' + userId + '/' + (isUserFollowed ? 'unfollow' : 'follow'), 
         data: {
             _token: csrfToken,
         },
-        success: function(response) {
-            button.text(action === 'follow' ? 'Unfollow' : 'Follow');
-            button.data('action', action === 'follow' ? 'unfollow' : 'follow');
-            button.toggleClass('btn-primary btn-danger');
+        success: function() {
+            button.text(isUserFollowed ? 'Follow' : 'Unfollow'); 
+            button.toggleClass('btn-primary btn-danger'); 
+
+            if (window.location.pathname === '/profile/' + userId) {
```

Comment:
I noticed that you guys always using manual concatenation. You can use ECMAScript Concatenation.